frontend/src/pages/projects/screens/detail/types.ts

```diff
@@ -0,0 +1,13 @@
+export enum ProjectSectionID {
+  WORKSPACE = 'data-science-workspaces',
+  STORAGE = 'storage',
+  DATA_CONNECTIONS = 'data-connections',
+  MODEL_SERVING = 'model-serving',
+}
+
+export enum ProjectSectionTitle {
```

Comment:
Why not make this a map to labels -- that way TypeScript can make sure we are looking at things correctly?

<img width="730" alt="image" src="https://user-images.githubusercontent.com/8126518/191353882-e9127f2e-e185-4a81-9099-ae3a0a1f60e3.png">

```
enum Test {
  ONE = 'alpha',
  TWO = 'beta',
}

type TestLabels = { [key in Test]: string };

const TestLabels: TestLabels = {
  [Test.ONE]: 'Alpha Label',
}

const alphaLabel = TestLabels[Test.ONE];
```